swap/swap.go

```diff
@@ -155,22 +153,90 @@ func (s *Swap) Add(amount int64, peer *protocols.Peer) (err error) {
 	log.Debug(fmt.Sprintf("balance for peer %s after accounting: %s", peer.ID().String(), strconv.FormatInt(peerBalance, 10)))
 
 	//check if balance with peer is over the payment threshold
-	if peerBalance >= s.paymentThreshold {
-		//if so, send cheque request message
-		log.Warn(fmt.Sprintf("balance for peer %s went over the payment threshold %v, requesting cheque", peer.ID().String(), s.paymentThreshold))
-		chequeRequestMessage := &ChequeRequestMsg{
-			Beneficiary: s.owner.address,
-		}
-		err = swapPeer.Send(context.TODO(), chequeRequestMessage)
-
+	if peerBalance <= -1*s.paymentThreshold {
```

Comment:
Please remove the multiplication. Just use `-s.paymentThreshold`

Reply:
Done.